shared/rakefiles/entrypoint.rake

```diff
@@ -108,6 +108,55 @@ task :display_cluster_info => [:set_vars] do
   puts
 end
 
+desc "Display gpii/universal image SHA and link to GitHub commit that triggered the build"
+task :display_universal_image_info => [:set_vars] do
+  sh "#{@exekube_cmd} sh -c ' \
```

Comment:
Consider setting `-e` flag to stop on errors? Or add some error handling, as there's none for most parts of this script - curl calls, an unexpected structure of outputs... (I think -e might be easy way out in this case ;))

Reply:
There are condition checks to handle most common failures (unable to get data from k8s, unable to generate commit link), but yeah, setting `-e` flag makes sense.

Reply:
Actually nevermind, it is not going to work properly with `set -e`.

Reply:
Fair enough, I think that it's generally dangerous to rely on output of given commands, not even checking for return codes. Like relying on empty output of `curl | jq` in a while condition.

Reply:
I see your point, but I think since this is just utility task, and nothing terrible going to happen even when it fails, current level of error handling is good enough.